hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -296,11 +296,10 @@
 
   public static final int OZONE_SCM_DEFAULT_PORT =
       OZONE_SCM_DATANODE_PORT_DEFAULT;
-  // File Name and path where datanode ID is to written to.
+  // The path where datanode ID is to be written to.
   // if this value is not set then container startup will fail.
-  public static final String OZONE_SCM_DATANODE_ID = "ozone.scm.datanode.id";
-
-  public static final String OZONE_SCM_DATANODE_ID_PATH_DEFAULT = "datanode.id";
+  public static final String OZONE_SCM_DATANODE_ID_DIR =
+      "ozone.scm.datanode.id";
```

Comment:
Can we add the "dir" part to the key string also. This is what would be used to configure this property in clusters.